src/ps.rs

```diff
@@ -6,22 +6,22 @@ use crate::jobs;
 use crate::nvidia;
 use crate::process;
 use crate::util::{three_places, time_iso8601};
+use std::collections::{HashMap, HashSet};
```

Comment:
The location of this is a stylistic issue and we should probably agree on how we like to do it (and as the module owner you get to decide :-) but I've been grouping `use` directives in blocks in the following order: `extern crate X`; `use crate X`; `use X::Y`, with directives for tests following that in the same order.  I'm not enough of a rust programmer to know what is "typical" or "standard" in the community; happy for feedback.

Reply:
I think this order makes sense. I will follow it.